Documentation/installation/k8s-install-helm.rst

```diff
@@ -163,8 +163,10 @@ Install Cilium
                      && iptables -t nat -F AWS-CONNMARK-CHAIN-1
 
          Some Linux distributions use a different interface naming convention.
-         If you use masquerading with the option ``egressMasqueradeInterfaces=eth0``,
-         remember to replace ``eth0`` with the proper interface name.
+         If you use masquerading with the option ``egressMasqueradeInterfaces=eth+``,
+         remember to replace ``eth+`` with the proper interface name. For
+         reference, Amazon Linux 2 uses ``eth+``, whereas Amazon Linux 2023 uses
+         ``ens+``.
```

Comment:
I think the docs should state that mixed node image families with different interface naming conventions is not currently supported.

Reply:
I've added a [quote](https://github.com/cilium/cilium/pull/36076/files#diff-13f16293a6e865c96800d64660bcd4bee961e7b8a60f4b2d4790249bdfe5152eR169).
Thanks!